File: Oke_server/oke/src/main/java/com/bnuz/oke/dto/CourseRecordData.java
package com.bnuz.oke.dto;

import com.bnuz.oke.entity.CourseRecord;

import java.io.Serializable;

/**
 * CourseRecordData
 *
 * @author handsome
 * @date 2020年 06月03日 16:17:50
 */
public class CourseRecordData implements Serializable {

	private static final long serialVersionUID = 3657770820939183637L;

	private CourseRecord courseRecord;

	private int score;

	public CourseRecord getCourseRecord() {
		return courseRecord;
	}

	public void setCourseRecord(CourseRecord courseRecord) {
		this.courseRecord = courseRecord;
	}

	public int getScore() {
		return score;
	}

	public void setScore(int score) {
		this.score = score;
	}

	@Override
	public String toString() {
		return "CourseRecordData{" +
				"courseRecord=" + courseRecord +
				", score=" + score +
				'}';
	}
}

File: Oke_client/Oke_teacher/Oke_teacher/Dto/CourseRecordData.cs
using Oke_teacher.Entity;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Oke_teacher.Dto
{
    /// <summary>
    /// 学生上课记录数据类
    /// </summary>
    [Serializable]
    public class CourseRecordData
    {
        //学生上课记录
        public CourseRecord courseRecord { set; get; }
        //学生总分
        public int score { set; get; }
    }
}
